doc/tools/coqrst/coqdomain.py

```diff
@@ -905,9 +905,17 @@ def is_coqtop_block(node):
         return isinstance(node, nodes.Element) and 'coqtop_options' in node
 
     @staticmethod
-    def split_sentences(source):
+    def split_sentences(node):
         """Split Coq sentences in source. Could be improved."""
-        return re.split(r"(?<=(?<!\.)\.)\s+", source)
+        lines = map(lambda s: s.rstrip(), node.rawsource.splitlines())
```

Comment:
IIUC, you want to split only at periods at end of line, correct? If so, it might be enough to `\s+` by `\n` in the regexp?

Reply:
You're right, I changed this.